package.json

```diff
@@ -69,6 +69,7 @@
   },
   "dependencies": {
     "@arbitrum/nitro-contracts": "1.3.0",
-    "@openzeppelin/contracts": "4.7.3"
+    "@openzeppelin/contracts": "4.7.3",
+    "fetch-logs-with-cache": "^0.0.11"
```

Comment:
moved to dev dependencies in the OP PR, but this under my personal NPM account at the moment